PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+            document.add_paragraph()
+    else:
+        pass
+
+    if ile == 1:
+        document.add_paragraph('Przewód może być użytkowany w parametrach określonych przez zamawiającego.')
+    else:
+        document.add_paragraph('Przewody mogą być użytkowane w parametrach określonych przez zamawiającego.')
+    document.add_paragraph()
+
+    if ile == 1:
+        uzytkowanie1 = document.add_paragraph('Użytkowanie przewodu wg stanu technicznego - biorąc pod uwagę mechaniczne zużycie, ewentualne przetarcia podczas użytkowania, stopień ryzyka dla obsługi i wszystkie inne czynniki mogące mieć wpływ na zastosowanie i żywotność przewodów.')
+        uzytkowanie1.alignment = WD_PARAGRAPH_ALIGNMENT.JUSTIFY # @UndefinedVariable
+    else:
+        uzytkowanie1 = document.add_paragraph('Użytkowanie przewodów wg stanu technicznego - biorąc pod uwagę mechaniczne zużycie, ewentualne przetarcia podczas użytkowania, stopień ryzyka dla obsługi i wszystkie inne czynniki mogące mieć wpływ na zastosowanie i żywotność przewodów.')
+        uzytkowanie1.alignment = WD_PARAGRAPH_ALIGNMENT.JUSTIFY # @UndefinedVariable
+    document.add_paragraph()
+
+    if ktoryoddzial.get() == '50':
+        przygzatw = document.add_paragraph('Przygotował:\t' + przygotowal + '\tZatwierdził:\t' + zatwierdzil)
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(13.75))
+
+        datydaty = document.add_paragraph('Data:\t' + kiedy_p + '\tData:\t' + kiedy_z)
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(13.75))
+
+        podpispodpis = document.add_paragraph('Podpis:\tPodpis:')
+        podpispodpis.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        podpispodpis.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+    else:
+        przygzatw = document.add_paragraph('Przygotował:\t' + przygotowal)
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+
+        datydaty = document.add_paragraph('Data:\t' + kiedy_p)
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+
+        podpispodpis = document.add_paragraph('Podpis:')
+
+    def filesave():
+        if ktoryoddzial.get() == '50':
+            sss = nQC
+        else:
+            sss = 'QC-'+ktoryoddzial.get()+'-'+nQC+'-'+strftime('%y')
+        filename = filedialog.asksaveasfilename(defaultextension='.docx', initialfile=sss)
+        if not filename: return
+        document.save(filename)
+        pytanieootwarcie = messagebox.askyesno("Co teraz?", "Gratuluję sukcesu!\nCzy chcesz teraz otworzyć zapisany dokument?", default="yes")
+        if pytanieootwarcie:
+            startfile(filename)
+        else:
+            pass
+    filesave()
+
+def generujpoangielsku():
```

Comment:
Jest taki przydatny wzorzec projektowy który nazywa się Strategia. Warto byłoby go zastosować do generowania. W skrócie tworzysz główną klasę `Generate` w której implementujesz wszystkie wspólne elementy generowania, niezależne od języka. Potem tworzysz dwie dziedziczące z niej klasy `GenerateEnglish` i `GeneratePolish', które reużywają kodu znajdującego się w klasie Generate oraz doprecyzowują i dopisują własne elementy, specyficzne dla języka.
https://sourcemaking.com/design_patterns/strategy tu jest przykład z fragmentem jak to powinno w Pythonie wyglądać.